_config.yaml

```diff
@@ -1,25 +1,33 @@
+title: CCDL Training
 training_title: MONTH-TOPIC
 # The name of the repository (e.g., 2020-may-training)
 # We use training-specific-template to help with development
 repository: training-specific-template
 # Full URL to the repository
 # Again, using training-specific-template to aid in development
-repository-url: https://github.com/AlexsLemonade/training-specific-template
-start-date: YYYY-MM-DD
-end-date: YYYY-MM-DD
+repository_url: https://github.com/AlexsLemonade/training-specific-template
+start_date: YYYY-MM-DD
+end_date: YYYY-MM-DD
 # Specific release in AlexsLemonade/training-modules
 # We use master to make development easier
-release-tag: master
+release_tag: master
 # The Docker user will almost always be ccdl, which is why this is here
-docker-user: ccdl
-docker-repo: DOCKER-REPOSITORY
-docker-tag: DOCKER-TAG
+docker_user: ccdl
+docker_repo: DOCKER-REPOSITORY
+docker_tag: DOCKER-TAG
 # These are for loading Docker images from a file; only relevant for in-person
-docker-targz: ccdl_docker_image.tar.gz
-docker-tar: ccdl_docker_image.tar
+docker_targz: ccdl_docker_image.tar.gz
+docker_tar: ccdl_docker_image.tar
 # The image ID and size can be obtained by running `docker images` once you have a copy of the correct image for the training
-docker-image-id: IMAGEID
-docker-size: SIZE
+docker_image_id: IMAGEID
+docker_size: SIZE
 # How we have historically distributed materials in-person
-training-zip: TRAINING-ZIP
-training-unzip-folder: TRAINING-UNZIP-FOLDER
+training_zip: TRAINING-ZIP
+training_unzip_folder: TRAINING-UNZIP-FOLDER
+
+# Theme and theme options
+theme: minima
+header_pages:
+  - virtual-workshop/SCHEDULE.md
+  - virtual-workshop/workshop-structure.md
+  - virtual-workshop/workshop-materials.md
```

Comment:
I know this is a draft, but let's make sure there's a note about how this is for development and should be changed to `workshop` (analogous to how we use `repository_url: https://github.com/AlexsLemonade/training-specific-template` and `repository: training-specific-template`).

Reply:
I was planning to change this before finishing, but I will also a documentation note.